### PR TITLE
Project review

### PR DESCRIPTION
**Pull request by medsonmoombe**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+  <meta charset="UTF-8">
+  <title>Survey form</title>
+  <link rel="stylesheet" href="style.css">
+
+</head>
```
Comment (medsonmoombe) [inline index.html:8]: Please add the viewport meta tag .The viewport meta tag allows you to tell the mobile browser what size this virtual viewport should be , vital for responsive layout.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    <label id="name-label" for="name" >
+   Name:
+  </label> <input id="name" type="text" placeholder="enter name" required>
+  <br>
+  <br>
+  <lable id="email-label" for="email" >email:</lable>
+    <input id="email" type="email" placeholder="enter email" required>
+    <br>
+    <br>
+    <lable id="number-label" for="number" >Number:</lable>
```
Comment (medsonmoombe) [inline index.html:24]: please kindly remove and minimize using break tags unnecessary in your code. It is problematic for people who navigate with the aid of screen reading technology.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    </div>
+   
+      <button type="submit" id="submit" class="submit-button">
+        Submit
+      </button>
+   
+    
+  </form>
+
+</body>
```
Comment (medsonmoombe) [inline index.html:134]: Please kindly avoid using a lot  div tags in your code and use the semantic HTML tags.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#main{
+    align-items: center;
+}
+body {
+    
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+
```
Comment (medsonmoombe) [inline style.css:10]: Please kindly include the link of the imported font family in your code.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+}
+#description{
+    color:chocolate;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-size: 25px;
+}
+input{
+    height: 30px;
+    width: 150px;
+}
```
Comment (medsonmoombe) [inline style.css:35]: Please make some changes and make sure the input boxes ,checkboxes and radios are inline with the texts and also reduce the margins between
changes
.

**Review by medsonmoombe — COMMENTED** [review]
Hi @medsonmoombe, 

Good job so far!
There are some issues that you still need to work on in your project but you are almost there!

## To Highlight please include the following
-  Set  up linters
- Your Readme file should be descriptive
- Add a proper descriptive 
- Add a .gitignore file

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @medsonmoombe, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**